homeassistant/components/elevenlabs/__init__.py

```diff
@@ -0,0 +1,56 @@
+"""The ElevenLabs text-to-speech integration."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from elevenlabs.client import AsyncElevenLabs
+from elevenlabs.core import ApiError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+
+from .const import CONF_MODEL, DEFAULT_MODEL
+from .tts import get_model_by_id
+
+PLATFORMS: list[Platform] = [Platform.TTS]
+
+
+@dataclass(kw_only=True, slots=True)
+class ElevenLabsData:
+    """ElevenLabs data type."""
+
+    client: AsyncElevenLabs
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Please extend the ConfigEntry type as displayed in https://developers.home-assistant.io/blog/2024/04/30/store-runtime-data-inside-config-entry